test/controllers/actions/RequiredUserAnswersForCorrectReturnSpec.scala

```diff
@@ -340,9 +343,8 @@ class RequiredUserAnswersForCorrectReturnSpec extends SpecBase with DefaultAwait
         "","","", UkAddress(List.empty, "", None),
         RetrievedActivity(smallProducer = true, largeProducer = true, contractPacker = true, importer = true, voluntaryRegistration = true),
         LocalDate.now(), List.empty, List.empty, Contact(None, None, "", ""), None)
-      val res = requiredUserAnswers.packingListReturnChange(basicUserAnswers, subscription)
-
-      res mustBe List.empty
+      val res = CorrectReturnBaseCYAPage.previousPagesRequired(basicUserAnswers, subscription)
+      res.filter(!_.additionalPreconditions.contains(false)).map(_.page).contains(PackAtBusinessAddressPage) mustBe false
```

Comment:
Could you please create this first part as it's own val and then add the desired page within the specific test?  It is repeated 3+ times and is a little hard to read because of the negative - so creating a val allows you to put a name that is clear like falsePagesFilteredOut or something,
res.filter(!_.additionalPreconditions.contains(false)).map(_.page).contains(